Assets/Scripts/main/Loader.cs

```diff
@@ -69,27 +71,50 @@ public static void LogData(TrialProgress s, long trialStartTime, Transform t, in
             {
                 using (var writer = new StreamWriter("Assets/OutputFiles~/" + DS.GetData().OutputFile, true))
                 {
-                    var trialIdStr = (s.TrialID + 1).ToString();
+                    var trialIdStr = "InTrial";
+                    var PositionX = t.position.x.ToString();
+                    var PositionZ = t.position.z.ToString();
+                    var RotationY = t.eulerAngles.y.ToString();
+                    var LoadingTime = 0.0;
+                    var timeSinceExperimentStart = DateTimeOffset.Now.ToUnixTimeMilliseconds() - DataSingleton.GetData().ExperimentStartTime;
+                    var timeSinceTrialStart = DateTimeOffset.Now.ToUnixTimeMilliseconds() - trialStartTime;
 
                     // If we're in a loading delay period we want to output that
                     // instead of TrialID;
                     if (s.isLoaded == false)
                     {
-                        trialIdStr = "Loading Delay";
+                        trialIdStr = "Loading";
+                        PositionX = "NA";
+                        PositionZ = "NA";
+                        RotationY = "NA";
+                        LoadingTime = DateTimeOffset.Now.ToUnixTimeMilliseconds() - trialStartTime;
                     }
 
-                    var timeSinceExperimentStart = DateTimeOffset.Now.ToUnixTimeMilliseconds() - DataSingleton.GetData().ExperimentStartTime;
-                    var timeSinceTrialStart = DateTimeOffset.Now.ToUnixTimeMilliseconds() - trialStartTime;
+                    if (s.Instructional == 1 & LoadingTime != 0)
+                    {
+                        PositionX = "NA";
+                        PositionZ = "NA";
+                        RotationY = "NA";
+                    }
+
+                    if(s.Instructional == 1)
+                    {
+                        PositionX = "NA";
+                        PositionZ = "NA";
+                        RotationY = "NA";
```

Comment:
This could probably be refactored into a separate function, not a bit deal for now though.